doc/nestml_language.md

```diff
@@ -391,7 +391,7 @@ end
 ### Block types
 
 * `neuron` *`<name>`* - The top-level block of a neuron model called `<name>`. The content will be translated into a single neuron model that can be instantiated in PyNEST using `nest.Create(<name>)`. All following blocks are contained in this block.
-* `parameters` - This block is composed of a list of variable declarations that are supposed to contain all variables which remain constant during the simulation, but can vary among different simulations or instantiations of the same neuron. These variables can be set and read by the user using `nest.SetStatus(<gid>, <variable>, <value>)` and  `nest.GetStatus(<gid>, <variable>)`.
+* `parameters` - This block is composed of a list of variable declarations that are supposed to contain all variables which remain constant during the simulation, but can vary among different simulations or instantiations of the same neuron. These variables can be set and read by the user using `node.set({<variable> : <value>})` and  `node.get(<variable>)`.
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#pet-peeves) there should be no space between the key and the colon in dictionary definitions. Please ignore this if you find this comment overly picky, or fix everywhere if you think it is justified.